packages/composer/test/openapi/routes-composition.test.js

```diff
@@ -245,7 +245,7 @@ test('should not compose api if there is no openapi nor graphql config', async (
       method: 'GET',
       url: '/api2/posts'
     })
-    assert.equal(statusCode, 404)
+    assert.equal(statusCode, 200)
```

Comment:
Can you flip this test back and send a different PR that just changes this? Seems this is breaking after all.

Reply:
Done, see: https://github.com/platformatic/platformatic/pull/2764